test/ejabberd_sm_SUITE.erl

```diff
@@ -58,9 +59,15 @@ tests() ->
      kv_can_be_updated_for_session,
      kv_can_be_removed_for_session,
      store_info_sends_message_to_the_session_owner,
-     remove_info_sends_message_to_the_session_owner
+     remove_info_sends_message_to_the_session_owner,
+     parse_session_key_s5_format,
+     parse_session_key_s4_format,
+     parse_session_key_s4_format_with_colon_in_sid
```

Comment:
I think we don't need to run these tests in each group, because they are redis-only, and we just perform the same exact test a few times.